docs/storage/s3/parquet.md

```diff
@@ -0,0 +1,38 @@
+# Apache Parquet
+
+There is idea to use Apache Parquet for storing group relations between Runtime and the Application.
+Apache Parquet is a column-oriented data storage format, which could be used to hold the needed pieces of information. It would be stored on the root of the S3 bucket. 
+
+### Pros
+
+1. **Efficient data store**
+
+    Very good compression. For example, if CSV file is as big as 1 TB, the data stored in Parquet file has 130 GB.
+
+1. **Good performance**
+    
+    Fetching specific column values doesn't need reading the entire raw data.
+
+### Cons
+
+1. **Doing queries**
+
+    For Golang, we have [parquet-go](https://github.com/xitongsys/parquet-go) library, which handles read and writes of the `parquet` file.
+
+    In order to do queries in Apache Parquet, we would need a query engine, such as [Apache Drill](https://drill.apache.org/), which is written in Java.
+
+    If we wanted to avoid this engine, in our use case, we would need to define group names and application names as columns.
+
+1. **Concurrent write into the same file**
+
+    We could enable bucket files versioning (for example, in [GCS](https://cloud.google.com/storage/docs/gsutil/addlhelp/ObjectVersioningandConcurrencyControl)). However, we would need to implement a mechanism to always fetch the latest version of the file and retry saving until success.
```

Comment:
It's not an option, that's one of the requirement that we need to keep previous revisions of APIs, docs etc

Reply:
In parquet file we wouldn't store APIs, docs, etc. - just info about all labels, and labels for all runtimes and apps